setup.py

```diff
@@ -15,7 +15,7 @@
 
 
 install_requires = (
-    'Django>=1.11,<4',
```

Comment:
Should we replace 4 with a 5 ??    Django 5 branch should be available around the end of 2023

Reply:
@ibiternas done in #f5e9e43